pom.xml

```diff
@@ -88,7 +88,7 @@
         <dependency>
             <groupId>com.graphql-java</groupId>
             <artifactId>graphql-java</artifactId>
-            <version>12.0</version>
+            <version>15.0</version>
```

Comment:
@jexp Do you think we need to add a compatibility layer to support older versions of `graphql-java`. Since they are not very compatible code my break. E.g. I'm using [graphql-spqr-spring-boot-starter](https://github.com/leangen/graphql-spqr-spring-boot-starter) with this library, which has not yet been updated to use version 15.0 of `graphql-java`. There is an open issue for this https://github.com/leangen/graphql-spqr/issues/356 . So also other users my have this compatibility issues.

Reply:
That was similar to the position I was in and why I tried to help update the library here. I wanted to use a library similar to spqr (graphql-kotlin) which uses a later graphql-java. Rather than a compatibility layer, I opted to help move this project forward with the thinking that a rising tide lifts all boats. I could help update other projects too. This is a common problem with libraries that use common dependencies. It is often an issue with Guava or something similar. The two strategies I've seen most often used successfully are to severely limit dependencies used in a library and/or to be aggressive about getting everyone to update the common deps.

Reply:
My suggestion is:
1. we release a version of 1.0.1 with the nested sorting PR merged and graphql-java 12 (and create a branch/tag for that)
2. then we release 1.1.0 with the upgrade of graphql-java to 15 (master)

Reply:
:+1: